internal/types/secret.go

```diff
@@ -361,5 +424,10 @@ func unredactedURL(new, old string) (string, error) {
 		newURL.User = url.User(newURL.User.Username())
 	}
 
-	return newURL.String(), nil
+	var urlCh bool
+	if newURL.Host != oldURL.Host {
+		urlCh = true
+	}
+
+	return newURL.String(), urlCh, nil
```

Comment:
nit: small simplification

```suggestion
	urlChanged := newURL.Host != oldURL.Host
	return newURL.String(), urlChanged, nil
```